schema-source/RS0003.schema.yaml

```diff
@@ -66,6 +66,9 @@ ComponentType:
     ELECTRIC_RESISTANCE_ELEMENT:
       Display Text: "Electric Resistance Element"
       Description: "Electric resistance heater elements"
+    DIRECT_EVAPORATIVE:
+      Display Text: "Direct Evaporative"
+      Description: "Wetted media cooling"
```

Comment:
"Wetted evaporative cooling media"?

Reply:
I changed the wording - none of the wording sounds quite right though!